test/unit/io/alignment_file/format_bam_test.cpp

```diff
@@ -446,8 +446,8 @@ TEST_F(bam_format, too_long_cigar_string_write)
 {
     // create an alignment resulting more than 65535 cigar elements
     // -------------------------------------------------------------------------
-    auto read = view::repeat_n('T'_dna5,  70'000);
-    auto ref  = view::repeat_n('A'_dna5, 2 * read.size() - 1);
+    auto read = views::repeat_n('T'_dna5,  70'000);
```

Comment:
missing include?

Reply:
We have a lot of missing indirect includes. Maybe we should try to tackle them for snippets, but I don't think it's important for tests.

Reply:
Also entirely unrelated to this PR...